src/index.html

```diff
@@ -3,8 +3,47 @@
   <head>
     <meta charset="utf-8"/>
     <title>Tax Calculator</title>
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
     <div>Static div test</div>
+    <div class="tbl">
+      <table>
```

Comment:
Please don't use `table` element here. It is not exactly a table even if it looks like one. Check this discussion https://stackoverflow.com/questions/83073/why-not-use-tables-for-layout-in-html